Assets/Demo/SimpleController_UsingActions_InAsset.cs

```diff
@@ -80,10 +104,20 @@ private void Move(Vector2 direction)
 
     private void Look(Vector2 rotate)
     {
-        var scaledRoateSpeed = rotateSpeed * Time.deltaTime;
-        m_Rotation.y += rotate.x * scaledRoateSpeed;
-        m_Rotation.x = Mathf.Clamp(m_Rotation.x - rotate.y * scaledRoateSpeed, -89, 89);
-        transform.localEulerAngles = m_Rotation;
+        //var scaledRoateSpeed = rotateSpeed * Time.deltaTime;
+        //m_Rotation.y += rotate.x * scaledRoateSpeed;
+        //m_Rotation.x = Mathf.Clamp(m_Rotation.x - rotate.y * scaledRoateSpeed, -89, 89);
+        //transform.localEulerAngles = m_Rotation;
```

Comment:
No commented out code without cause please. Either add comment why code is being kept or remove commented out code.